pkgs/development/haskell-modules/generic-builder.nix

```diff
@@ -53,6 +53,8 @@
 , coreSetup ? false # Use only core packages to build Setup.hs.
 , useCpphs ? false
 , hardeningDisable ? lib.optional (ghc.isHaLVM or false) "all"
+, hasDataDir ? true
+, hasDocDir ? true
```

Comment:
What exactly makes a Haskell build have a doc dir? Am I correct in assuming that this equivalent to `doHaddock || extra-doc-files /= []`?